### PR TITLE
Add case property history toggle

### PR DESCRIPTION
**Pull request by proteusvacuum**
This should hopefully just be temporary... 👿 
https://manage.dimagi.com/default.asp?265989
@emord @sheelio

### CODE REVIEW

**Review by emord — APPROVED** [review]

File: corehq/toggles.py [hunk 1]
```diff
@@ -923,6 +923,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
     help_link='https://confluence.dimagi.com/display/ccinternal/Support+Flag',
 )
 
+CASE_PROPERTY_HISTORY = StaticToggle(
+    'case_property_history',
+    'Shows a modal on the case property page allowing you to see the history of various case properties',
+    TAG_SOLUTIONS,
```
Comment (emord) [inline corehq/toggles.py]: Err. Actually can you add the namespace domain here?

**Review by emord — CHANGES_REQUESTED** [review]

File: corehq/toggles.py (hunk 1 above)
Comment (emord) [reply corehq/toggles.py]: Also I think if this is going to get released, it shouldn't be solutions? @sheelio would know more

Comment (sheelio) [reply corehq/toggles.py]: @proteusvacuum 
Ya, I would product path it if we do intend to release it.

Comment (proteusvacuum) [reply corehq/toggles.py]: Updated

**Review by emord — APPROVED** [review]

**Pull request merged**